src/interfaces/modular/GaussianProcess_includes.i

```diff
@@ -25,7 +25,7 @@
 
  #include <shogun/machine/gp/InferenceMethod.h>
  #include <shogun/machine/gp/LaplacianInferenceBase.h>
- #include <shogun/machine/gp/FITCInferenceBase.h>
+ #include <shogun/machine/gp/SparseInferenceBase.h>
```

Comment:
For the future. Pls do the renaming in a different patch, this is very  hard to review otherwise